examples/connectivity/plot_adhd_covariance.py

```diff
@@ -5,8 +5,8 @@
 This example shows how to extract signals from regions defined by an atlas,
 and to estimate a covariance matrix based on these signals.
 """
- 
-plotted_subject = 0  # subject to plot
+n_subjects = 10  # # subjects to consider for group-sparse covariance (max: 40)
```

Comment:
OK.  I think many of the comments in examples are (or should be) duplicated in docstrings, that the goal was to help newbies get up to speed faster and play with the parameters in scripts.

At least, that's definitely how I'm using them :)

Will remove the second "#".  I converted "Number of" to "#", for space reasons, but it can just be eliminated!  Let me know if you feel strongly about the "max" thing, I won't push any harder than my words above :).